content/docs/user-guide/editor/customizing.md

```diff
@@ -85,58 +76,51 @@ You can change the general O3DE Editor settings and file settings.
 | Show Time in Console |  Displays the time in the console window.  |
 | Toolbar Icon Size |  Adjusts the toolbar icon size.  Default: `16` (32 pixels).  |
 | Stylus Mode |  Enables stylus mode for tablets and other pointing devices.  |
-|  **Enable UI 2.0 (EXPERIMENTAL)**  |  Enables the updated user interface (UI) in O3DE Editor.  |
-|  **Enable Scene Inspector (EXPERIMENTAL)**  | Enables the option to inspect scenes in files such as .fbx files. |
-|  **Restore Viewport Camera on Game Mode Exit**  |  Returns the camera to the original transform when you exit gameplay mode.  |
-|  **Enable Legacy UI (DEPRECATED)**  |  Enable the legacy user interface (UI) in O3DE Editor.  |
-|  **Enable New Viewport Interaction Model (EXPERIMENTAL)**  |  Replaces the old interaction model in the viewport. For more information, see [Working with the Viewport Interaction Model](/docs/user-guide/editor/viewport/).  |
-
+| Enable UI 2.0 (EXPERIMENTAL) |  Enables the updated user interface (UI) in O3DE Editor.  |
```

Comment:
update: Should we remove the name UI 2.0 now (as it's the default and definitely not experimental anymore (tagging @AMZN-daimini and @cgalvan for visibility).

Reply:
I didn't update the pages in the Editor topics outside of condensing the pages and some minor formatting, this/these topics definitely need a full update. I've created an issue for documenting Editor settings: https://github.com/o3de/o3de.org/issues/2076.  

I will make some quick corrections to this table right now though.

Reply:
That would be awesome thanks @FiniteStateGit 👍